fabtests/common/shared.c

```diff
@@ -3216,6 +3217,37 @@ void ft_parse_opts_range(char* optarg)
 	test_size = range_test_size;
 }
 
+void ft_parse_opts_list(char* optarg)
+{
+	int i, ret;
+	char *token;
+
+	optarg += 2; // remove 'l:'
+	test_cnt = 1;
+	for (i=0; optarg[i] != '\0'; i++) {
+		test_cnt += optarg[i] == ',';
```

Comment:
It would be annoying for someone with a sticky keyboard, e.g. `1,,2,,3,`

Reply:
It will raise error during parsing and print `sscanf cannot parse integer in list`

Reply:
Will it?  strtok() will skip over multiple tokens, and we don't verify that test_cnt integers were read.  I'm not actually worried about this.  Worse case, we have some 0 size values in the array, if it doesn't error out.

Reply:
Ah, I didn't realize that was strtok's behavior, but of course you are correct.

Your solution with test_cnt as loop variable is a good one, I did that, and tested to confirm.

I also noticed sscanf doesn't typically set errno, so perror may not be appropriate.  Testing a list with some garbage values gave me:
`sscanf cannot parse integer in list: Success` so I changed the perror to a fprintf(stderr,...).